src/components/pages/HomePage.jsx

```diff
@@ -48,43 +45,43 @@ const HomePage = () => {
             backgroundImage: `url(${GreenBlobImage})`,
           }}
         >
-          <p className='font-spartan font-semibold text-white text-2xl lg:text-3xl w-1/5 sm:w-1/3'>
+          <p className="font-spartan font-semibold text-white text-2xl lg:text-3xl w-1/5 sm:w-1/3">
             Courses for every age
           </p>
         </div>
         <div
           className="w-full h-60 sm:h-80 md:h-96  bg-contain bg-center bg-no-repeat flex justify-center items-center"
           style={{ backgroundImage: `url(${YellowBlobImage})` }}
         >
-          <p className='font-spartan font-semibold text-white text-2xl lg:text-3xl w-1/5 sm:w-1/3'>
+          <p className="font-spartan font-semibold text-white text-2xl lg:text-3xl w-1/5 sm:w-1/3">
             Over 100 subject taught
           </p>
         </div>
         <div
           className="w-full h-60 sm:h-80 md:h-96 bg-contain bg-center bg-no-repeat flex justify-center items-center"
           style={{ backgroundImage: `url(${PurpleBlobImage})` }}
         >
-          <p className='font-spartan font-semibold text-white text-2xl lg:text-3xl w-1/5 sm:w-1/3'>
+          <p className="font-spartan font-semibold text-white text-2xl lg:text-3xl w-1/5 sm:w-1/3">
             1000+ experienced teachers
           </p>
         </div>
       </section>
       <section className="px-4 py-8 flex justify-center">
-          <div className="grid grid-cols-1 sm:grid-cols-2 md:grid-cols-3 gap-12 md:w-10/12">
-            {items.map((item, index) => (
-              <div
-                key={index}
-                className="flex items-center text-center rounded-lg"
-              >
-                <img src={item.src} alt={item.alt} className="mr-2" />
-                <p className="text-xl md:text-2xl font-spartan font-medium hover:text-darkGreen hover:cursor-pointer">
-                  {item.text}
-                </p>
-              </div>
-            ))}
-          </div>
+        <div className="grid grid-cols-1 sm:grid-cols-2 md:grid-cols-3 gap-12 md:w-10/12">
+          {items.map((item, index) => (
+            <div
+              key={index}
```

Comment:
Careful about using index as a key. If the array is change it could cause issues with how things are rendered. Using an external library like UUID would work much better. If you are sure that no changes will be made that this is ok for now.